frontend/src/components/SearchPage.js

```diff
@@ -4,28 +4,76 @@ import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
 import { DropdownButton, SearchBarCustom } from './'
 import { updateField } from '../redux/modules/searchpage'
+import { updateResults } from '../redux/modules/results'
+import axios from 'axios'
+import { Button } from 'reactstrap'
 
-const mapStateToProps = state => ({})
+const mapStateToProps = state => ({
+  system: state.searchpage.consoles,
+  age: state.searchpage.ageRange,
+  symptom: state.searchpage.symptoms,
+  name: state.searchpage.nameSearchField
+})
 
 const mapDispatchToProps = dispatch => {
   return bindActionCreators(
     {
-      updateField
+      updateField,
+      updateResults
     },
     dispatch
   )
 }
 
+function getGames(age, symptom, system) {
+  return axios
+    .get(
+      'http://localhost:8080/games' + '?age=' + age + '&symptom=' + symptom + '&system=' + system
+    )
+    .then(response => {
+      console.log(
+        'http://localhost:8080/games' + '?age=' + age + '&symptom=' + symptom + '&system=' + system
+      )
+      return response.data.result.games
+    })
+    .catch(function(error) {
+      console.log('ERROR: ', error)
+      return null
+    })
+}
+
```

Comment:
A couple things here: First, lets put this method in a separate file which can be used to wrap all API calls. Second, make `http://localhost:8080` a constant string in that file so that once the backend is deployed, we can change this link in one place, rather than in many.